packages/@okta/vuepress-site/docs/concepts/sign-in-widget/index.md

```diff
@@ -0,0 +1,75 @@
+
+#### Standard
+
+Use the standard sign-in form hosted at your unique Okta URL for a no-code implementation. You can upload your own company logo, a background image, and a favicon. You can also define a primary and secondary color palette. You can also change the labels and text for various elements in the Sign-In Widget, such as headings, labels, and links. This includes customizing the **Username** field to specify what identifiers users can use, or adding a custom help link.
+
+The standard sign-in form also allows you to customize the sign-in experience for each app or website. For example, if you have multiple brands with distinct websites and domain names under a parent company or you need a different sign-in experience for various apps, you can use distinct branding for each. See [Branding](https://help.okta.com/oie/en-us/content/topics/settings/branding.htm).
+
+#### Advanced
+If you've configured a custom domain and need more control over the Sign-In Widget's appearance, use the built-in code editor. You can make more advanced changes to the HTML, CSS, and JavaScript of the sign-in form. For the third-generation widget, you can use design tokens to customize elements like colors, fonts, borders, and spacing, ensuring consistency with your brand's style guide. See [Style the Sign-In Widget (third generation)](https://developer.okta.com/docs/guides/custom-widget-gen3/main/) or [Style for redirect authentication (second generation)](https://developer.okta.com/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget).
+
+
+### Self-hosted Sign-In Widget
+
+Embed the Sign-In Widget into your own code base to host the authentication client on your servers. Use this approach if you need the highest level of customization and modifications to the open-source code to fit your requirements. Since the authentication client is hosted on your servers, security is a shared responsibility and you must update Sign-In Widget packages. You can specify the version of the Sign-In Widget to use, either by referencing it from a CDN or installing it locally through npm. This helps to manage updates and ensure stability.
+
+When you initialize the Sign-In Widget in your code, you pass a configuration object. This object allows you to specify various options:
+
+* `logoText` and `brandName`: These options display your brand's name and logo.
+
+* `helpSupportNumber`: This option displays a support number for password reset and account unlock flows.
+
+* `i18n` (Internationalization): This option overrides default text strings and labels for different languages.
+
+See [Embedded Okta Sign-In Widget fundamentals](https://developer.okta.com/docs/guides/embedded-siw/main/).
```

Comment:
We should probably reference the sign-in widget repo here as well: https://github.com/okta/okta-signin-widget